tests/integration/components/post-view-test.js

```diff
@@ -0,0 +1,21 @@
+import hbs from "htmlbars-inline-precompile";
+import { moduleForComponent, test } from "ember-qunit";
+
+moduleForComponent("post-view", {
+  integration: true
+});
+
+test("renders the post", function(assert) {
+  let post = { title: "Test post", contentHtml: "hello world!" };
+  this.set("post", post);
+  this.render(hbs`{{post-view post=post}}`);
```

Comment:
Expected ')' and instead saw '{{post-view post=post}}'.<br>Missing semicolon.<br>Expected an identifier and instead saw ')'.<br>Expected an assignment or function call and instead saw an expression.